fastlane/Fastfile

```diff
@@ -373,6 +383,25 @@ end
     end
   end
 
+  #############################################################################
+  # trigger_app_store_ci_build
+  # ---------------------------------------------------------------------------
+  # This lane triggers a build for App Store distribution on CI
+  # ---------------------------------------------------------------------------
+  # Usage:
+  # bundle exec fastlane trigger_release_build [branch_to_build:<branch_name>]
+  #
+  #############################################################################
+  lane :trigger_app_store_ci_build do | options |
```

Comment:
I thought about making this a `private_lane`, but I think it's useful to be able to call it manually if we need to.

What do you think?